draft-ietf-tls-tls13.md

```diff
@@ -2194,9 +2194,8 @@ server sends an empty extension to indicate negotiation of this
 extension and the OCSP information is carried in a CertificateStatus
 message. In TLS 1.3, the server's OCSP information is
```

Comment:
Might be worth noting that the v1 status_request is forbidden inline here, e.g., "In TLS 1.3, the RFC 6066 status_request extension is disallowed, so only the RFC 6961 status_request_v2 may be used, and the server's OCSP information is carried in an extension in EncryptedExtensions".